### PR TITLE
Add event-handler/ to deploy scripts

### PR DESCRIPTION
**Pull request by cshcomcom**

### CODE REVIEW

File: deploy_blockchain_genesis_gcp.sh [hunk 1]
```diff
@@ -139,7 +139,7 @@ function inject_account() {
 
 # deploy files
 FILES_FOR_TRACKER="blockchain/ block-pool/ client/ common/ consensus/ db/ blockchain-configs/ logger/ tracker-server/ traffic/ package.json setup_blockchain_ubuntu.sh start_tracker_genesis_gcp.sh start_tracker_incremental_gcp.sh"
-FILES_FOR_NODE="blockchain/ block-pool/ client/ common/ consensus/ db/ blockchain-configs/ json_rpc/ logger/ node/ p2p/ tools/ traffic/ tx-pool/ package.json $KEYSTORE_DIR setup_blockchain_ubuntu.sh start_node_genesis_gcp.sh start_node_incremental_gcp.sh wait_until_node_sync_gcp.sh"
+FILES_FOR_NODE="blockchain/ block-pool/ client/ common/ consensus/ db/ event-handler/ blockchain-configs/ json_rpc/ logger/ node/ p2p/ tools/ traffic/ tx-pool/ package.json $KEYSTORE_DIR setup_blockchain_ubuntu.sh start_node_genesis_gcp.sh start_node_incremental_gcp.sh wait_until_node_sync_gcp.sh"
```
Comment (platfowner) [inline deploy_blockchain_genesis_gcp.sh:142]: could you move blockchain-configs/ backward to make it sorted?

File: deploy_blockchain_incremental_gcp.sh [hunk 2]
```diff
@@ -121,7 +121,7 @@ elif [[ $ACCOUNT_INJECTION_OPTION = "--mnemonic" ]]; then
 fi
 
 FILES_FOR_TRACKER="blockchain/ block-pool/ client/ common/ consensus/ db/ blockchain-configs/ logger/ tracker-server/ traffic/ package.json setup_blockchain_ubuntu.sh start_tracker_genesis_gcp.sh start_tracker_incremental_gcp.sh"
-FILES_FOR_NODE="blockchain/ block-pool/ client/ common/ consensus/ db/ blockchain-configs/ json_rpc/ logger/ node/ p2p/ tools/ traffic/ tx-pool/ $KEYSTORE_DIR package.json setup_blockchain_ubuntu.sh start_node_genesis_gcp.sh start_node_incremental_gcp.sh wait_until_node_sync_gcp.sh"
+FILES_FOR_NODE="blockchain/ block-pool/ client/ common/ consensus/ db/ event-handler/ blockchain-configs/ json_rpc/ logger/ node/ p2p/ tools/ traffic/ tx-pool/ $KEYSTORE_DIR package.json setup_blockchain_ubuntu.sh start_node_genesis_gcp.sh start_node_incremental_gcp.sh wait_until_node_sync_gcp.sh"
```
Comment (platfowner) [inline deploy_blockchain_incremental_gcp.sh:124]: ditto

**Review by platfowner — APPROVED** [review]
LGTM with some minor comments.
Thanks!

File: deploy_blockchain_genesis_gcp.sh (hunk 1 above)
Comment (cshcomcom) [reply deploy_blockchain_genesis_gcp.sh]: Thanks! I moved `blockchain-configs/` to between `blockchain/` and `block-pool/`

File: deploy_blockchain_incremental_gcp.sh (hunk 2 above)
Comment (cshcomcom) [reply deploy_blockchain_incremental_gcp.sh]: Done

**Comment by cshcomcom** [comment]
Thanks for the quick review! Now merging

**Pull request merged**